content/en/docs/concepts/services-networking/ingress-controllers.md

```diff
@@ -29,6 +29,7 @@ Kubernetes as a project supports and maintains [AWS](https://github.com/kubernet
 * The [Citrix ingress controller](https://github.com/citrix/citrix-k8s-ingress-controller#readme) works with
   Citrix Application Delivery Controller.
 * [Contour](https://projectcontour.io/) is an [Envoy](https://www.envoyproxy.io/) based ingress controller.
+* [EnRoute](https://getenroute.io) is an open source Ingress Controller API Gateway based on [Envoy](https://www.envoyproxy.io) that includes support to automate OpenAPI spec ingestion and advanced rate-limiting.
```

Comment:
```suggestion
* [EnRoute](https://getenroute.io/) is an [Envoy](https://www.envoyproxy.io) based API
  gateway that can act as an ingress controller.
```
How about this wording?

Reply:
Except for one small word change, it looks good to me. Does this one sound reasonable? -

[EnRoute](https://getenroute.io/) is an [Envoy](https://www.envoyproxy.io) based API gateway that can run as an ingress controller.

Reply:
@sftim Pushed a commit with the changes. Thanks !